.github/workflows/conan-package.yml

```diff
@@ -0,0 +1,152 @@
+      - '.github/workflows/conan-package.yml'
+    branches:
+      - main
+      - master
+      - dev
+      - 'release/**'
+
+env:
+  CONAN_USER: ${{ secrets.CONAN_USER }}
+  CONAN_PASS: ${{ secrets.CONAN_PASS }}
+
+jobs:
+  get-semver:
+
+    runs-on: ubuntu-20.04
+
+    outputs:
+      semver_full: ${{ steps.git-tool.outputs.fullSemVer }}
+      semver_majorminorpatch: ${{ steps.git-tool.outputs.MajorMinorPatch }}
+      user: ${{ steps.get-conan-broadcast-data.outputs.user }}
+      channel: ${{ steps.get-conan-broadcast-data.outputs.channel }}
+
+    steps:
+      - name: Checkout CuraEngine
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
+
+      - name: Install GitVersion
+        uses: gittools/actions/gitversion/setup@v0.9.13
+        with:
+          versionSpec: '5.x'
+
+      - id: git-tool
+        name: GitTools
+        uses: gittools/actions/gitversion/execute@v0.9.13
+
+      - id: get-conan-broadcast-data
+        name: Get Conan broadcast data
+        run: |
+          echo ${{ github.repository_owner }} | awk '{print "::set-output name=user::"tolower($0)}'
+          
+          if [ "${{ github.ref_name == 'master' || github.ref_name == 'main' }}" = "true" ]; then
+            echo ${{ github.repository_owner }} | awk '{print "::set-output name=user::"tolower($0)}'
+            echo '::set-output name=channel::stable'
+          elif [ "${{ startsWith(github.ref_name, 'release/') || steps.git-tool.outputs.fullSemVer == steps.git-tool.outputs.MajorMinorPatch }}" = "true" ]; then
+            echo '::set-output name=user::_'
+            echo '::set-output name=channel::_'
+          elif [ "${{ github.ref_name == 'dev' || github.event_name == 'merge' || github.event_name == 'pull_request' }}" = "true" ]; then
+            echo ${{ github.repository_owner }} | awk '{print "::set-output name=user::"tolower($0)}'
+            echo '::set-output name=channel::testing'
+          else
+            echo ${{ github.repository_owner }} | awk '{print "::set-output name=user::"tolower($0)}'
+            echo ${{ github.ref_name }} | awk '{print "::set-output name=channel::"substr(tolower($0),0,9)}'
+          fi
+
+  conan-package:
+    runs-on: ${{ matrix.os }}
+
+    needs: [ get-semver ]
+
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ macos-10.15, windows-2022, ubuntu-20.04]
+
+    steps:
+      - name: Checkout CuraEngine
+        uses: actions/checkout@v3
+
+      - name: Setup Python and pip
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10.4'
+          architecture: 'x64'
+          cache: 'pip'
+
+      - name: Prepare Conan and sip-build (Bash)
+        if: ${{ runner.os != 'Windows' }}
+        run: |
+          pip install --require-hashes -r requirements.txt
+          conan config install https://github.com/Ultimaker/conan-config.git -a "-b CURA-9177_Fix_CI_CD" -tf $HOME/.conan
```

Comment:
Not really a big fan of hardcoding a branch here. I'd argue that this should either be main or it should be it's own repo. Maybe it should even try to check out the same branch name as the current branch so we can create PR's that do depend on different conan config automagically.

Reply:
Hard coding the branch of the Conan config was added to ensure our current builds keep using the same old conan-config and that my changes don't mess these up. When merged and completely enabled in the new Conan process through all our  repo's then should be simple `conan config install https::github.com/ultimkaer/conan-config.git'
I will add a TODO to remind us